components/CheckoutWizard/types.ts

```diff
@@ -1,3 +1,3 @@
-export type CheckoutViewProps = {
-  assetId: string | null
+export type AssetIdProp = {
+  assetId: string | undefined
```

Comment:
when defining a type shape you can use `property?: type` to infer that it's either `type` or `undefined`

Reply:
True that.